Makefile

```diff
@@ -1,14 +1,21 @@
+bootstrap: check-postgres postgres-mm webkit-snapshot-mm init-db xcodeproj
```

Comment:
Keeping our default `make` command first makes it easy to link to should we want to.